app/views/books/_book.html.erb

```diff
@@ -6,6 +6,5 @@
         <td><%= @book_types.key(book.book_type_id) %></td>
         <td><%= book.price %></td>
         <td><%= book.status %></td>
-    <th>
```

Comment:
check lại indent của file nhé
c thấy cái th kia của chú với cái td đang cùng cấp
mà tại sao cái trên th cái dưới lại td thế?

Reply:
tại em muốn in đậm cái id để phân biệt rõ các dòng